libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -938,6 +944,11 @@ void _IotMqtt_DecrementConnectionReferences( _mqttConnection_t * pMqttConnection
     ( pMqttConnection->references )--;
     IotMqtt_Assert( pMqttConnection->references >= 0 );
 
+    /* In some implementations IotLog() maps to C standard printing API 
+     * that need specific primitive types for format specifiers. Also, 
+     * inttypes.h may not available on some C99 compilers, despite stdint.h 
```

Comment:
Needs a grammar fix, this appears throughout so please search/replace – "Also, inttypes.h may not be available on some C99 compilers..."

Reply:
It is done. I will merge, I would like to squash commits.